### ISSUE TITLE
SQLSTATE[HY000]: General error: 7 number of parameters must be between 0 and 65535

### ISSUE THREAD

**Issue by boboldehampsink** [putyourlightson/craft-blitz#639]
### Bug Report

Seeing this come up in various variants:

```
SQLSTATE[HY000]: General error: 7 number of parameters must be between 0 and 65535 The SQL being executed was: SELECT "id" FROM "blitz_caches" WHERE (("siteId"=1) AND ("uri"='...')) OR (("siteId"=1) AND ("uri"='......
```

I'm on PostgreSQL

### Plugin Version

4.13

### Craft CMS Version

4.8

### PHP Version

8.3

**Comment by bencroker** [comment]
Are you seeing a stack trace along with these errors, or can you track one down in the logs?

**Comment by boboldehampsink** [comment]
Ah, forgot to mention this is shown as a "Refreshing Blitz Cache" job error (no full stacktrace comes with that)

**Comment by bencroker** [comment]
Thanks! Fixed in https://github.com/putyourlightson/craft-blitz/commit/2440fd8c81c67fd1b38bc468016e923073cdcc62 for the next release.

**Comment by boboldehampsink** [comment]
Thanks! When is the release planned? My queue manager is actually crashing now a few of these big ones have shown up - have to delete them manually from the db to make things work again.

**Comment by bencroker** [comment]
Early next week. If you need this sooner then you can install the `develop` branch by running `ddev composer require "putyourlightson/craft-blitz:dev-develop as 4.13.3"`.

**Comment by boboldehampsink** [comment]
@bencroker now seeing this slightly different message (using develop):

```
SQLSTATE[HY000]: General error: 7 number of parameters must be between 0 and 65535 The SQL being executed was: SELECT "siteId", "uri" FROM "blitz_caches" WHERE "id" IN (205807, 205908, 182419, ...
```

**Comment by bencroker** [comment]
Thanks. I’ve added chunking of query params in more places in https://github.com/putyourlightson/craft-blitz/commit/4c0edf674afab2ff9b9b18eb0d27616d618c3fcc. Hopefully that catches them all now.

**Comment by bencroker** [comment]
Released in [4.14.0](https://github.com/putyourlightson/craft-blitz/releases/tag/4.14.0).
